index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Huseyin Biyik's Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="menu-container">
+            <a class="logo-container" href="#">
+                <p>Huseyin Biyik</p>
+            </a>
+            <nav>
+                <a href="#" class="hamburger-menu">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </a>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <div class="tagline-container">
+            <div class="primary-text-container">
+                <h1>
+                    Huseyin Biyik
+                </h1>
+            </div>
+            <div class="supporting-text-container">
+                <h2>
+                    Software Developer
+                </h2>
+            </div>
```

Comment:
Kindly try to fix the style between your divs. use `margin top>`  between `<div class="tagline-containe>` and `<div supporting-text-container>`, so i invite you to have fun with your divs to explore the magic of CSS while referring to this [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) that was proposed in the project requirements